qooxdoo-kit/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+set -e
+
+if [ "$1" = 'help' ]; then
+    cat << 'HELLO_END'
+Welcome to the OSPARC Qooxdoo Kit
+---------------------------------
+a qooxdoo compiler and a copy of the qooxdoo sdk.
+
+Use the compiler in the same way as you would use it if you
+had a local installation.
+
+ docker run -it -v $(pwd):/project itisfoundation/qx-kit:latest qx create myapp -t desktop -I
+ docker run --init -it -v $(pwd)/myapp:/project itisfoundation/qx-kit:latest qx compile
+ docker run --init -it -v $(pwd)/myapp:/project itisfoundation/qx-kit:latest -P 8080:8080 qx serve
+
+HELLO_END
+exit 1
+fi
+
+PROJECT_DIR=/project
+
+export PATH=/home/node/node_modules/.bin:$PATH
+USERID=$(stat -c %u ${PROJECT_DIR})
```

Comment:
We normally check that ``/project`` (in your case) is mounted and report an error. Just for clarity

Besides, if you define ``VOLUME /project`` in your ``Dockerfile``, it will unfortunately be owned by the user building the container

Reply:
not defining VOLUME :)

Reply:
added a check for project